R/utils.R

```diff
@@ -14,6 +14,13 @@ risk_category_levels <- function() {
   c("high", "medium", "low")
 }
 
+#' Categorize variable into risk categories using low and high thresholds.
+#'
+#' @param x variable.
+#' @param low_threshold Dataframe column.
+#' @param high_threshold Dataframe column.
```

Comment:
```suggestion
```

Reply:
This is a thin wrapper around `dplyr::case_when()`, and very similar to `ifelse()` -- so the thresholds can but don't have to be columns.

``` r
tiltIndicator:::categorize_risk(c(1:30), 10, 20)
#>  [1] "low"    "low"    "low"    "low"    "low"    "low"    "low"    "low"   
#>  [9] "low"    "low"    "medium" "medium" "medium" "medium" "medium" "medium"
#> [17] "medium" "medium" "medium" "medium" "high"   "high"   "high"   "high"  
#> [25] "high"   "high"   "high"   "high"   "high"   "high"
```


Also I suggest removing the documentation. As a developer-oriented function we expect the user to be able to read source code. This makes it easier to maintain.

Else I would document it in full -- adding for example documentation for `...` and also examples.

Reply:
Makes sense! I have removed the suggested documentation. Thanks for the review!